### DISCUSSION TITLE
The wood does not break

### DISCUSSION THREAD

**Discussion by timoschwark** [Royal-Modding/FS19_Firewood#7] [Q&A]
Good day, I have since a few days that I can no longer break the wood with my axe on a server. This does not break and is not stored in a pallet. However, this is close enough. I also do not get any errors displayed. With my friend the mod works normally, he also plays on the server. I have already once reinstalled the mod and also the entire game, but in vain. Likewise, I have already deleted all orders of the game and recreate. All in vain. It just does not work anymore. We have already rebuilt the server and also reinstalled the mod on it. This really only does not work for me. Even in single player I can no longer do this. I now do not know what to do and need help. Possibly one of you already had the problem, or a modder could help me. 

Thanks for your efforts in advance.

I speak English and German, but can also translate other languages, thank you!

**Comment by fcelsa** [comment]
Hi, after your descriptions, one things that I have in mind: it's the input bindings problem, double check those and generate a new one in the case.

**Comment by timoschwark** [comment]
Hello, thank you for your new answer. How can I change this then? I use the normal keyboard and mouse.

Von: Fabio Celsa ***@***.***>
Gesendet: Mittwoch, 19. Mai 2021 10:08
An: Royal-Modding/FS19_Firewood ***@***.***>
Cc: timoschwark ***@***.***>; Author ***@***.***>
Betreff: Re: [Royal-Modding/FS19_Firewood] The wood does not break (#7)


Hi, after your descriptions, one things that I have in mind: it's the input bindings problem, double check those and generate a new one in the case.

**Reply by fcelsa** [reply]
The best way it's to delete file inputBindings.xml in the farming simulator user folder, where is the mods directory; in the next start the game will create a clean new one.
Another way it's to check in the options from main menu, after game start, before load a career savegame.

**Comment by timoschwark** [comment]
Thanks a lot, I'll give it a try and let you know.

Von: Fabio Celsa ***@***.***>
Gesendet: Mittwoch, 19. Mai 2021 11:24
An: Royal-Modding/FS19_Firewood ***@***.***>
Cc: timoschwark ***@***.***>; Author ***@***.***>
Betreff: Re: [Royal-Modding/FS19_Firewood] The wood does not break (#7)


The best way it's to delete file inputBindings.xml in the farming simulator user folder, where is the mods directory; in the next start the game will create a clean new one.
Another way it's to check in the options from main menu, after game start, before load a career savegame.

**Comment by timoschwark** [comment]
Hello, unfortunately this did not work either, would you have other options?

Von: Fabio Celsa ***@***.***>
Gesendet: Mittwoch, 19. Mai 2021 11:24
An: Royal-Modding/FS19_Firewood ***@***.***>
Cc: timoschwark ***@***.***>; Author ***@***.***>
Betreff: Re: [Royal-Modding/FS19_Firewood] The wood does not break (#7)


The best way it's to delete file inputBindings.xml in the farming simulator user folder, where is the mods directory; in the next start the game will create a clean new one.
Another way it's to check in the options from main menu, after game start, before load a career savegame.
